### PR TITLE
69 authprovider and sessionprovider

### LINKED ISSUE #69: AuthProvider and SessionProvider

### PR DESCRIPTION
**Pull request by yongcho-tx**
AuthProvicder has been created and wrapped around main app in 'layout.tsx'.

### CODE REVIEW

**Comment by yongcho-tx** [comment]
Let me know what you think.

File: context/AuthProvider.tsx
Comment (SeanPaulson) [inline context/AuthProvider.tsx:1]: Looks good! only suggestion is to change this AuthProvider.tsx file to 
use client'

```
import { SessionProvider } from "next-auth/react"

export default SessionProvider;
```
This way you don't have to worry about the types.

**Review by SeanPaulson — CHANGES_REQUESTED** [review]
To use nextAuth generated types.

File: context/AuthProvider.tsx
Comment (yongcho-tx) [reply context/AuthProvider.tsx:1]: Changed as suggested.

**Review by SeanPaulson — APPROVED** [review]

**Pull request merged**